### ISSUE TITLE
Redshift interval

### ISSUE THREAD

**Issue by OlivierKauffmann** [JADES-GTO/JAGUAR_extragalactic_mock_catalog#6]
The redshift interval of the galaxy catalog is actually 0.2<z<15. How was the value of 0.2 chosen ? In the analogous work Schreiber+2017, they do not go up to z=0 as well (z>0.05) and explain that this is done not to have very bright extended galaxies in the catalog.

**Comment by kevinhainline** [comment]
Our catalog was designed to combine the results from luminosity and mass functions from the literature to constrain our number counts and galaxy properties. The mass functions we used ([Tomczak et al. 2014](http://adsabs.harvard.edu/abs/2014ApJ...783...85T)) are only defined at z > 0.2 due to the redshift limits on the surveys that were used to measure them. As this catalog was primarily designed to explore the properties of galaxies at higher redshifts for the JADES survey, we chose to only probe down to this redshift (the volume of space at z = 0.0 - 0.2 in our survey was low enough that we aren't expecting a huge number of very low redshift galaxies). Was there anything you were particularly aiming to do at z < 0.2?

**Comment by OlivierKauffmann** [comment]
I am performing source detection simulations to compute the purity of high-redshift samples given an observation strategy. In this context, the low-redshift contaminants are important, that’s why I expected to use a complete galaxy sample with redshifts in 0<z<15. As you said, the number of objects at z<0.2 will be small, but they should be included or at least quantified.

Even though you used empirical mass functions defined at z>0.2, you built a continuous model for the mass function whose parameters are defined for z>0. It sounds reasonable to me to extrapolate to redshifts below z=0.2, although this extrapolation might not be very accurate. What do you think ?

**Comment by kevinhainline** [comment]
Yeah, because we have no constraints between z = 0 - 0.2, I don't know if I'd use our formalism for extrapolating to those low redshifts. Both the star-forming and quiescent mass functions blow up significantly as you go to low redshifts, especially at the low-mass end. I don't know if I'd trust the accuracy of these number counts for your project.
